recipes/hole2/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hole2" %}
+{% set version = "2.3.2" %}
+{% set build_number = environ.get('GIT_DESCRIBE_NUMBER', '0') %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/MDAnalysis/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz"
+  sha256: 67225d2134dc7a14a01c23b30022a0fc00d23f8adc1e5d617da330509ce64576
```

Comment:
Will this be changed to https://github.com/osmart/hole2/ as soon as we make a release https://github.com/osmart/hole2/issues/5 ?

Reply:
Yes